documentation/en/curio-pdp.md

```diff
@@ -0,0 +1,105 @@
+---
+description: This page explains how to setup a Curio PDP SP
+---
+
+# Curio PDP
+
+This guide is focused on running a PDP-only SP.  Most of the same instructions will apply for running a curio node providing both PoRep and PDP storage though you will likely need to modify the configuration defaults specified here to keep the sealing pipeline running smoothly.
+
+## Setup curio 
+
+The first step to running a curio PDP SP is to install and setup curio.
+
+Some important callouts departing from the ordinary curio setup instructions
+* You MUST check out `feat/pdp` branch and build curio from source on this branch. PDP code is not yet on `main`
+* The `feat/pdp` branch has not been updated to go v1.24.x so running PDP SP requires building with go v1.23.x
```

Comment:
This basically means, users cannot build on calibnet or mainnet. We should call that out.

Reply:
I think it should still work with Go v1.23.7, but this needs confirmation from @ZenGround0 which is currently running the Calibnet Curio PDP SP displayed here: https://calibration.pdp-explorer.eng.filoz.org/

So maybe we only need to write that Go v1.23.7 is the only version you can be on at the moment

Reply:
Oh! My bad. I misread this. You can ignore this one.